### ISSUE TITLE
[debian packaging] apt changelog fails

### ISSUE THREAD

**Issue by rexbron** [jellyfin/jellyfin#3017]
**Describe the bug**
`$ apt changelog jellyfin

E: Failed to fetch changelog:/jellyfin.changelog  Changelog unavailable for jellyfin=10.5.5-1
` 

**System (please complete the following information):**
 - OS: Debian 10
 - Virtualization: None
 - Jellyfin Version: 10.5.5-1

**To Reproduce**

1. run `apt changelog jellyfin`

**Expected behavior**

A listing of the changelog for the debian package.

**Comment by telans** [comment]
Pretty sure this is fixed on nightlies

**Comment by joshuaboniface** [comment]
The release changelogs simply link to the GitHub release pages. They should work though, but it seems both stable and unstable packages don't list the changelog. This doesn't make much sense since they're definitely included, unless there's something wrong with our `dpkg-buildpackage` command.

**Comment by joshuaboniface** [comment]
The changelog file is definitely there, e.g.

```
$ zcat /usr/share/doc/jellyfin-server/changelog.Debian.gz 
jellyfin-server (20200703.8-unstable) unstable; urgency=medium

  * Jellyfin Server unstable build 20200703.8 for merged PR #3495

 -- Jellyfin Packaging Team <packaging@jellyfin.org>  Fri, 03 Jul 2020 18:57:27 +0000
```

I'm not sure why `apt` and `apt-get` can't read them though.
